Assets/LeapMotion/Core/Scripts/DataStructures/Movement.cs

```diff
@@ -0,0 +1,106 @@
+﻿using UnityEngine;
+
+namespace Leap.Unity {
+
+  /// <summary>
+  /// A struct containing Vector3s for velocity and angularVelocity, useful
+  /// to encapsulate the derivative over time of a Pose struct.
+  ///
+  /// angularVelocity is always stored as an Angle-Axis vector, whose angle
+  /// amount is the vector's magnitude in degrees (not radians).
+  /// </summary>
+  public struct Movement {
+
+    /// <summary>
+    /// The linear velocity of this Movement.
+    /// </summary>
+    public Vector3 velocity;
+
+    /// <summary>
+    /// Angular velocity expressed as an angle-axis vector with angle equal to the length
+    /// of the vector in degrees.
+    /// </summary>
+    public Vector3 angularVelocity;
+
+    public static readonly Movement identity = new Movement();
+
+    public Movement inverse {
+      get { return new Movement(-velocity, -angularVelocity); }
+    }
+
+    public static Movement operator *(Movement movement, float multiplier) {
+      return new Movement(movement.velocity * multiplier,
+                          movement.angularVelocity * multiplier);
+    }
+
+    public static Movement operator /(Movement movement, float divisor) {
+      return movement * (1f / divisor);
+    }
+
+    public Pose ToPose() {
+      var angVelMag = angularVelocity.magnitude;
+      return new Pose(velocity,
+        Quaternion.AngleAxis(angVelMag, angularVelocity / angVelMag));
+    }
+
+    public static Movement operator +(Movement movement0, Movement movement1) {
+      return new Movement(movement0.velocity + movement1.velocity,
+                          movement0.angularVelocity + movement1.angularVelocity);
```

Comment:
Adding angular velocities this way is not actually a correct way to add angular velocities.  It is definitely convenient, but it might be worth noting somewhere.

Reply:
Will comment